.elasticbeanstalk/config.yml

```diff
@@ -0,0 +1,18 @@
+branch-defaults:
```

Comment:
Did you write this yml file? or was this generated by the system?

Reply:
This was generated by awsebcli when I ran the command eb init.